### ISSUE TITLE
Tooltip for Emoji & Stickers

### ISSUE THREAD

**Issue by s-light** [telegramdesktop/tdesktop#16206]
### Is your feature request related to a problem?
easily find the shortcode/name for a Emoji or Sticker

### Describe the solution you'd like
show a Tooltip on Emoji and Stickers with the name and shortcode

### Additional context
this would help to learn the text shortcodes and give the option to know what exactly was used... (sometimes its hard to find the same emoji)

moved  from [Enhancement: Emoji zoom - electrofloat commented on Sep 9, 2017](https://github.com/telegramdesktop/tdesktop/issues/2804#issuecomment-328281095)
> Tooltiping it with maybe the text version of the emoji would be indeed great. (explaining what that emoji is)

**Comment by 23rd** [comment]
Guess can be closed?

<img width="310" height="139" alt="Image" src="https://github.com/user-attachments/assets/b39c4771-38e5-4ea3-8ab8-7d2df30f991d" />

**Comment by s-light** [comment]
yes that is implemented - at least in my version i also can see it!
